pkg/controller/metrics/metrics.go

```diff
@@ -225,6 +235,25 @@ func (mc *Calculator) Start(ctx context.Context) error {
 						mcLog.Infof("cleared metric: %v", metricClusterDeploymentSyncsetPaused)
 					}
 				}
+
+				// Check if cluster is transitioning
+				hibernatingCond := controllerutils.FindClusterDeploymentCondition(cd.Status.Conditions, hivev1.ClusterHibernatingCondition)
+				readyCond := controllerutils.FindClusterDeploymentCondition(cd.Status.Conditions, hivev1.ClusterReadyCondition)
+				poolRef := ""
+				if cd.Spec.ClusterPoolRef != nil {
+					poolRef = cd.Spec.ClusterPoolRef.PoolName
```

Comment:
We need to account for the pool's namespace as well as its name. Nothing stopping you from having pools of the same name in different namespaces. For other pool-ish metrics ([example](https://github.com/openshift/hive/blob/374913322b5d94efa5ff0fdcd1a6055df351476f/pkg/controller/clusterpool/metrics.go#L13)) I've used separate labels for the namespace and name -- not sure if that's the best thing, or if we should combine them à la NamespacedName.